packages/uswds-core/src/styles/functions/color/get-system-color.scss

```diff
@@ -76,3 +78,10 @@ variable that is, itself, a list
 
   @return $output;
 }
+
+// @debug "It should accept quoted or unquoted color-value params. Expect: #2c608a";
+// @debug get-system-color("blue", 60);
+// @debug get-system-color("blue", "60");
+
+// @debug "It should correctly access vivid color values. Expect: #008659";
+// @debug get-system-color("mint", 50, "vivid");
```

Comment:
**note:** I originally added these lines to match some of the other debug tests in our other sass functions such as `is-color-dark()`

https://github.com/uswds/uswds/blob/ae7129cafeeff2089e9ccfd4b286c50e8d7d0b5f/packages/uswds-core/src/styles/functions/color/is-color-token.scss#L18-L19

I modified my approach to match our unit test styles since the sass `@return` is not usable outside of a sass function like these lines suggest and will not return the expected value.

**question:** Are these lines unnecessary with the addition of the sass unit tests? 

_lines added in this PR:_

https://github.com/uswds/uswds/blob/f1dec29d162863aa5b2bb31bdae621611ace16dc/packages/uswds-core/src/test/tests.scss#L657-L674